Assets/stylesheet.css

```diff
@@ -0,0 +1,131 @@
+ /* *{
+    -webkit-box-sizing: border-box;
+    -moz-box-sizing: border-box;
+    box-sizing: border-box;
+} */
+nav{
+    position: absolute;
+    left: 160px;
+    top: 85px;
+    width: 980px;
+    height: 36px;
+    display: inline-block;
+    border: #7a7a7a 1px solid;
+    border-radius: 4px;
+    background: linear-gradient(#737373 50%,#777777 25%,#5d5d5d 50% );
+}
+nav ul{
+    height: 36px;
+    padding: 0;
+    margin-top: 0;
+}
+nav a{
+    text-shadow: 2px 2px #363636;
+    text-decoration: none;
+    color: white;
+}
+nav li{
+    display: flex;
```

Comment:
`REQUIRED`

Let's add `hover` effects to the `li` elements of the navbar. See image below:

|Current|Preffered|
| :-----: | :------: |
| ![current apple nav hover](https://user-images.githubusercontent.com/36057474/62106589-7e4b8980-b29d-11e9-8249-6afff62e582c.gif)  | ![preffered apple nav hover](https://user-images.githubusercontent.com/36057474/62106599-84da0100-b29d-11e9-929c-9fd802900bb5.gif)  |